views/partials/header.hbs

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
Head should be a separate partial and header partial should have header,, logo and header navigation of the page.